src/pages/mentor-home/MentorHome.js

```diff
@@ -4,15 +4,14 @@ import BecomeATutor from '~/containers/mentor-home-page/become-a-tutor/BecomeATu
 import { ModalContext } from '~/context/modal-context'
 
 const MentorHome = () => {
```

Comment:
It should have margins from body on x-axis 32px on y-axis 24px.

Reply:
changed, added paperProps to modal